### DISCUSSION TITLE
Java Minecraft not blocked by HidHide

### DISCUSSION THREAD

**Discussion by vrs99-hub** [nefarius/HidHide#47] [Q&A] [answered]
I was kind of surprised that HidHide has no effect on [Minecraft controller mods](https://www.curseforge.com/minecraft/mc-mods/controllable?page=2#comments) to see my connected controllers. The mods seems to make use of [GLFW](https://www.glfw.org/). Is that a library that is unaffected by HidHide?

![Minecraft HidHide](https://user-images.githubusercontent.com/90147299/133474263-a1f5d7e5-4c88-40db-90a4-74b54e620637.png)

**Comment by nefarius** [comment, marked as answer]
See #39 XInput hiding via the UI Client is currently broken for not yet addressed issues.
